src/index.js

```diff
@@ -16,27 +16,6 @@ module.exports = function (context, options) {
     getThemePath() {
       return path.resolve(__dirname, './theme');
     },
-    configureWebpack(config) {
-      const generatedFilesDir = config.resolve.alias['@generated']
-      languages = utils.generateLunrClientJS(generatedFilesDir, options.languages);
-      // Ensure that algolia docsearch css is its own chunk
```

Comment:
This will break the lanuage support

Reply:
Dry I applyed the fix for my repo. Should be oy first commit. I will fix the PR tomorrow

Reply:
Sure